tests/pytest.ini

```diff
@@ -4,3 +4,5 @@ filterwarnings =
     error
     ignore:Creating an ndarray from ragged nested sequences
     ignore:`np.bool` is a deprecated alias for the builtin `bool`. To silence this warning, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
+    ignore:SelectableGroups dict interface is deprecated. Use select.:DeprecationWarning
+    ignore:tostring\(\) is deprecated. Use tobytes\(\) instead.:DeprecationWarning
```

Comment:
Where do these two deprecations happen? I am running all the tests with the latest versions of the libraries in ubuntu and Python 3.10 and they where not prompted,

Reply:
Python 3.7